### PR TITLE
ZENKO-513 feat: backbeat dashboard

### PR DESCRIPTION
**Pull request by giacope**
This depends on: https://github.com/scality/backbeat/pull/410

### CODE REVIEW

**Comment by giacope** [comment]
@bert-e wait

File: kubernetes/zenko/charts/backbeat/dashboards/backbeat.json [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+        "y": 0
+      },
+      "id": 6,
+      "panels": [],
+      "title": "CRR",
+      "type": "row"
+    },
+    {
+      "aliasColors": {
+        "crr_number_of_sites{app=\"backbeat-api\",chart=\"backbeat-1.0.0-RC5\",heritage=\"Tiller\",instance=\"10.1.14.56:8900\",job=\"kubernetes-service-endpoints\",kubernetes_name=\"zenko-backbeat-api\",kubernetes_namespace=\"default\",release=\"zenko\"}": "#806eb7"
```
Comment (philipyoo) [inline kubernetes/zenko/charts/backbeat/dashboards/backbeat.json:66]: will `instance` ip vary here? Also `kubernetes_name` depending on deployment name?

Also, if possible, maybe use variables for things like `chart`

Comment (giacope) [reply kubernetes/zenko/charts/backbeat/dashboards/backbeat.json:66]: These fields don't really matter. I will delete them.

**Review by ssalaues — APPROVED** [review]
LGTM good finds on the bad port numbers

**Review by philipyoo — APPROVED** [review]

**Comment by giacope** [comment]
@ssalaues and @philipyoo closing this in favor of #297

**Pull request closed without merging**